app/services/outgoings_creation_service.rb

```diff
@@ -0,0 +1,33 @@
+class OutgoingsCreationService < BaseCreationService
+  attr_reader :params
+
+  def initialize(params)
+    @params = params
+  end
+
+  def call
+    self
+  end
+
+  def as_json(_options = nil)
+    {
+      success: success?,
+      outgoings: (outgoings if success?),
+      errors: error_message
+    }
+  end
+
+  def outgoings
+    @outgoings ||= Outgoing.transaction do
+      params.map { |attrs| Outgoing.create(attrs) }
+    end
```

Comment:
These outgoings don't belong to any assessment?
I think you need to get the `assessment` and then you should be able to do `assessment.outgoings.create!(params)`

Reply:
Good point - thank you.